tests/ruleToggler.js

```diff
@@ -0,0 +1,162 @@
+var path = require('path'),
+    fs = require('fs'),
+    groot = require('../lib/groot'),
+    ruleToggler = require('../lib/ruleToggler'),
+    assert = require('assert'),
+    deepEqual = require('deep-equal');
+
+var getToggledRules = ruleToggler.getToggledRules,
+    isResultEnabled = ruleToggler.isResultEnabled;
+
+var generateToggledRules = function (filename) {
+  var filePath = path.join(process.cwd(), 'tests', 'sass', filename);
+  var file = {
+    'text': fs.readFileSync(filePath),
+    'format': path.extname(filePath).replace('.', ''),
+    'filename': path.basename(filePath)
+  };
+  var ast = groot(file.text, file.format, file.filename);
+  return getToggledRules(ast);
+};
+
+describe.only('rule toggling', function () {
```

Comment:
Be sure to remove this `.only` so that Coveralls will report an accurate coverage.